### PR TITLE
chore: add kotlin/swift repos to mgmt

### PR DESCRIPTION
**Pull request by toddbaert**
- adds kotlin-sdk to org management
- add swift-sdk to org management
- adds contributors from spotify

Marked as draft until we have the right people identified.

@fabriziodemaria @Calibretto @vahidlazio @nickybondarenko please add recommend maintainers and approvers. Keep in mind they will need to already be org members. Let me know if there's anyone else to invite, and keep in mind our charters and code of conduct, https://github.com/open-feature/community/blob/main/README.md.

I will keep myself as a maintainer for the time being to help get things setup, like CI, etc. Long term I may not continue in the role since (at least for now) I'm not too familiar with iOS or Android development.

### CODE REVIEW

File: config/open-feature/sdk-kotlin/workgroup.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+repos:
+  - kotlin-sdk
+
+approvers:
+
+maintainers:
+  - toddbaert
+
```
Comment (toddbaert) [inline config/open-feature/sdk-kotlin/workgroup.yaml:8]:
```suggestion
```

File: config/open-feature/sdk-kotlin/workgroup.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+repos:
+  - kotlin-sdk
+
+approvers:
+
+maintainers:
+  - toddbaert
```
Comment (toddbaert) [inline config/open-feature/sdk-kotlin/workgroup.yaml:7]: Please add the appropriate people or a `[]`

File: config/open-feature/sdk-swift/workgroup.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+repos:
+  - swift-sdk
+
+approvers:
+
+maintainers:
+  - toddbaert
```
Comment (toddbaert) [inline config/open-feature/sdk-swift/workgroup.yaml:7]: Please add the appropriate people or a `[]`

**Review by fabriziodemaria — COMMENTED** [review]
Added the suggestions with the list of maintainers

File: config/open-feature/sdk-kotlin/workgroup.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+repos:
+  - kotlin-sdk
+
+approvers:
```
Comment (toddbaert) [inline config/open-feature/sdk-kotlin/workgroup.yaml:4]:
```suggestion
approvers: []
```

File: config/open-feature/sdk-swift/workgroup.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+repos:
+  - swift-sdk
+
+approvers:
```
Comment (toddbaert) [inline config/open-feature/sdk-swift/workgroup.yaml:4]:
```suggestion
approvers: []
```

**Comment by toddbaert** [comment]
@fabriziodemaria I've applied your suggestions and moved from draft. Please note that `approvers` is empty. Approvers don't have quite as many privileges as maintainers, but can approve PRs and do triage. You can add others here in the future if you'd like.

Not the CI failure is expected - [it doesn't cope well with new group definitions](https://github.com/open-feature/community/pull/221#issuecomment-1623064769).

**Review by fabriziodemaria — APPROVED** [review]

**Review by nickybondarenko — APPROVED** [review]

**Review by alina-v1 — APPROVED** [review]

**Review by Calibretto — APPROVED** [review]

**Review by beeme1mr — APPROVED** [review]

**Pull request merged**